src/www/ui_components/feedback-view.html

```diff
@@ -172,6 +174,14 @@
       _computeShouldShowLanguageDisclaimer: function(hasEnteredEmail) {
         return !window.navigator.language.match(/^en/i) && hasEnteredEmail;
       },
+      _submitFeedback: function(event) {
+        const params = {
+          bubbles: true,
+          composed: true,
+          detail: this.getValidatedFormData(),
+        };
+        this.dispatchEvent(new CustomEvent("SubmitFeedbackRequested", params));
```

Comment:
Don't you need `bubbles: true, composed: true`?
Can you verify that the event handler is running?